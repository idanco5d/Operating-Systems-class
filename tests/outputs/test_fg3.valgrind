==6263== Memcheck, a memory error detector
==6263== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==6263== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==6263== Command: /mnt/c/Users/illus/os/hw/1/smash
==6263== Parent PID: 6258
==6263== 
==6263== 
==6263== FILE DESCRIPTORS: 4 open (3 std) at exit.
==6263== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_fg3.valgrind
==6263==    <inherited from parent>
==6263== 
==6263== 
==6263== HEAP SUMMARY:
==6263==     in use at exit: 0 bytes in 0 blocks
==6263==   total heap usage: 143 allocs, 143 frees, 85,851 bytes allocated
==6263== 
==6263== All heap blocks were freed -- no leaks are possible
==6263== 
==6263== For lists of detected and suppressed errors, rerun with: -s
==6263== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
