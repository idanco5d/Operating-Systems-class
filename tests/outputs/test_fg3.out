smash> smash> smash> smash> smash> smash> sleep 300& : 9058
smash: got ctrl-C
smash: process 9058 was killed
smash> [1] sleep 100& : 9053 1 secs
[2] sleep 200& : 9056 1 secs
[4] sleep 400& : 9061 1 secs
[5] sleep 500& : 9063 1 secs
smash> sleep 100& : 9053
smash: got ctrl-C
smash: process 9053 was killed
smash> [2] sleep 200& : 9056 1 secs
[4] sleep 400& : 9061 1 secs
[5] sleep 500& : 9063 1 secs
smash> sleep 200& : 9056
smash: got ctrl-C
smash: process 9056 was killed
smash> [4] sleep 400& : 9061 1 secs
[5] sleep 500& : 9063 1 secs
smash> sleep 500& : 9063
smash: got ctrl-C
smash: process 9063 was killed
smash> [4] sleep 400& : 9061 2 secs
smash> sleep 400& : 9061
smash: got ctrl-C
smash: process 9061 was killed
smash> smash> smash: sending SIGKILL signal to 0 jobs:
